cmd/dex/serve.go

```diff
@@ -91,6 +91,9 @@ func runServe(options serveOptions) error {
 	if err != nil {
 		return fmt.Errorf("invalid config: %v", err)
 	}
+
+	logger.Infof("dex version: %s", version)
```

Comment:
What do you think about printing Go runtime-specific versions along with the Dex version like here?https://github.com/dexidp/dex/blob/f70015dfedfba1d2dda0031f2f8c46d3d56260fd/cmd/dex/version.go#L18-L23

Reply:
I dont have any problem in adding them. Please let me know if you want me to add them as part of this PR.
@sagikazarmark

Reply:
In my opinion, it might be helpful.

Reply:
sure, I will add that :)

Reply:
@nabokihms I updated the PR. Please check if it looks good to you.

Reply:
Great!